lib/MwbExporter/Formatter/Doctrine2/Annotation/OnCorps/ApiPlatformFieldAnnotations.php

```diff
@@ -10,18 +10,24 @@
 
 use MwbExporter\Formatter\Doctrine2\Model;
 
+/**
+ * Class ApiPlatformFieldAnnotations
+ * @package MwbExporter\Formatter\Doctrine2\Annotation\OnCorps
+ */
 abstract class ApiPlatformFieldAnnotations
 {
-
     /**
      * @param Model\Table $table
+     * @param string      $fieldName
      *
-     * @return $this
+     * @return string
      */
-    abstract function buildAnnotations(Model\Table $table);
+    abstract function buildAnnotationProperty(Model\Table $table, Model\Column $column, string $type): string;
```

Comment:
for better readability can I suggest we use a `use` statement at the top of the file for ...`Model\Table` so we can use `Table` ?

Reply:
I'd like to keep this as is please.

Reply:
ok, just mentioning that the suggested usage of use statement here is a common standardised pattern in the wider php community and it is something that we actively flagged in other OnCorps php codebase as I think it is part of a psr coding style standard, I think phpstorm auto-import in fact works based on that.